src/js/components/agency/visualizations/obligated/ObligatedVisualization.jsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * ObligatedVisualization.jsx
+ * Created by Lizzie Salita 6/8/17
+ */
+
+import React from 'react';
+import moment from 'moment';
+import { convertQuarterToDate } from 'helpers/fiscalYearHelper';
+import * as MoneyFormatter from 'helpers/moneyFormatter';
+import _ from 'lodash';
+
+import AgencyObligatedGraph from './ObligatedGraph';
+
+const propTypes = {
+    activeFY: React.PropTypes.number,
+    reportingFiscalQuarter: React.PropTypes.number,
+    agencyName: React.PropTypes.string,
+    obligatedAmount: React.PropTypes.number,
+    budgetAuthority: React.PropTypes.number
+};
+
+export default class AgencyObligatedAmount extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            windowWidth: 0,
+            visualizationWidth: 0
+        };
+
+        this.handleWindowResize = _.throttle(this.handleWindowResize.bind(this), 50);
+    }
+
+    componentDidMount() {
+        this.handleWindowResize();
+        window.addEventListener('resize', this.handleWindowResize);
+    }
+
+    componentWillUnmount() {
+        window.removeEventListener('resize', this.handleWindowResize);
+    }
+
+    handleWindowResize() {
+        // determine if the width changed
+        const windowWidth = window.innerWidth;
+        if (this.state.windowWidth !== windowWidth) {
+            // width changed, update the visualization width
+            this.setState({
+                windowWidth,
+                visualizationWidth: this.sectionHr.offsetWidth
+            });
+        }
+    }
+
+    render() {
+        const obligatedValue = this.props.obligatedAmount;
+        const authorityValue = this.props.budgetAuthority;
+
+        const authUnits = MoneyFormatter.calculateUnitForSingleValue(authorityValue);
+        const authority = `${MoneyFormatter.formatMoney(authorityValue / authUnits.unit)}
+        ${authUnits.longLabel}`;
+
+        const obUnits = MoneyFormatter.calculateUnitForSingleValue(obligatedValue);
+
+        const amountObligated = `${MoneyFormatter.formatMoney(obligatedValue / obUnits.unit)}
+        ${obUnits.longLabel}`;
+
+        const endOfQuarter = convertQuarterToDate(this.props.reportingFiscalQuarter, this.props.activeFY);
+        const asOfDate = moment(endOfQuarter).format("MMMM D, YYYY");
```

Comment:
When parsing date strings, it is best to provide the [format of the input string](http://momentjs.com/docs/#/parsing/string-format/) to avoid ambiguity over months/days.